doc/source/troubleshoot/index.rst

```diff
@@ -15,6 +15,33 @@ some of the most common problems and frequently asked questions are posted here.
    faq
 
 
+Debug in PyMAPDL
+----------------
+
+If you are having trouble with PyMAPDL, you can examine the content
+of the output file to help to identify any issue. 
+
+You can set the logger output file to be ``mylog.log`` by
+running the following commands in a Python terminal or at the beginning of your
+script:
+
+.. code:: python
+
+    from ansys.mapdl.core import LOG
+    LOG.setLevel("DEBUG")
+    LOG.log_to_file("mylog.log")
+
+    from ansys.mapdl.core import launch_mapdl
+
+    mapdl = launch_mapdl(loglevel="DEBUG")
+
+You can attach this file to a bug report in the PyMAPDL GitHub repository for further investigation.
+If you are not able to identify the issue, you can open a discussion in
+`PyMAPDL - Discussions <pymapdl_discussions_>`_.
+If you believe you have found a bug, please open an issue in the
+`PyMAPDL GitHub repository <pymapdl_issues_>`_.
```

Comment:
```suggestion
You can attach this file to a bug report in the PyMAPDL GitHub repository for further investigation.
If you are not able to identify the issue, you can open a discussion on the
`PyMAPDL Discussions page <pymapdl_discussions_>`_.
If you believe you have found a bug, open an issue on the
`PyMAPDL Issues page <pymapdl_issues_>`_.
```
This is how I've been referring to them (as pages). However, I see that is not what's being used in line 51 and line 54. I guess it's only important to be consistent within the topic!